packages/main/src/plugin/openshell-cli/openshell-cli.spec.ts

```
@@ -0,0 +1,558 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { RunError, RunResult } from '@openkaiden/api';
+import { beforeEach, describe, expect, test, vi } from 'vitest';
+
+import type { CliToolRegistry } from '/@/plugin/cli-tool-registry.js';
+import type { Proxy } from '/@/plugin/proxy.js';
```

Biome (2.4.16)

[error] Do not shadow the global "Proxy" property.

(lint/suspicious/noShadowRestrictedNames)